p2p/nat/nat.go

```diff
@@ -333,13 +336,23 @@ func (nat *NAT) NewMapping(maddr ma.Multiaddr) (Mapping, error) {
 
 func (nat *NAT) establishMapping(m *mapping) {
 	oldport := m.ExternalPort()
+
 	log.Debugf("Attempting port map: %s/%d", m.Protocol(), m.InternalPort())
-	newport, err := nat.nat.AddPortMapping(m.Protocol(), m.InternalPort(), "http", MappingDuration)
+	comment := "libp2p"
+	if m.comment != "" {
```

Comment:
Ah, i see. I had assumed the comment field came from a prexisting NAT mapping on a router